ios-sdk/about.mdx

```diff
@@ -0,0 +1,51 @@
+---
+id: about
+title: Overview
```

Comment:
The title should be something like `Apple iOS SDK (or something more explaining)` so it's correctly indexed on google.